### ISSUE TITLE
gdb select.c no such file or directory

### ISSUE THREAD

**Issue by juppytt** [doppt/data-oriented-attacks#1]
Hi. I followed your instructions, but got stuck while running `sudo ./create_addrmap.sh`.
It says ../sysdeps/unix/sysv/linux/select.c: No such file or directory.
Is this an environmental issue? like proftpd is not compatiable with my linux version..

**Comment by juppytt** [comment]
I tried this again in Hong Hu's DOP Virtual machine.
I rebuilt proftpd with debug symbols, and `./create_addrmap.sh` finds necessary address information. However, it seems there is a problem in perl instruction. It can't search any address information from raw_addr_map.rb file.
Anyway I manually set addresses in addr_map.rb according to the generated raw_addr_map.rb. 
An then ran the following scripts (there also was an error in update_msf_script.sh that there is no .orig file. I just commented out that line.) 
```
sudo ./create_bytemap.sh 
sudo ./update_msf_script.sh 
run_exploit.sh
```
But the metasploit results 
`Exploit failed: EOFError EOFError`
